metadata/src/main/java/org/apache/kafka/metadata/migration/KRaftMigrationDriver.java

```diff
@@ -134,8 +139,7 @@ private void initializeMigrationState() {
     }
 
     private boolean isControllerQuorumReadyForMigration() {
-        // TODO implement this
-        return true;
+        return this.apiVersions.isAllNodeZkMigrationReady();
```

Comment:
When we check to see if the quorum nodes are ready, it would be useful to print an error log showing which controllers were not ready here (similar to what we do in `areZkBrokersReadyForMigration`).

Reply:
Good suggestion. Updated. Thanks.